custom-namespaces.yaml

```diff
@@ -1028,16 +1028,29 @@ data_integrity_monitoring:
   views:
     run_history:
       tables:
-        - table: data-monitoring-dev.dim.dim_run_history
+        - table: data-monitoring-dev.dim_external.run_history
       type: table_view
     run_processing_info:
       tables:
-        - table: data-monitoring-dev.dim.dim_run_processing_info
+        - table: data-monitoring-dev.dim_external.run_processing_info
       type: table_view
     muted_alerts:
       tables:
-        - table: data-monitoring-dev.dim.dim_muted_alerts
+        - table: data-monitoring-dev.dim_external.muted_alerts
       type: table_view
+  explores:
+    run_history:
+      type: explore
```

Comment:
Is this a valid explore type?

Reply:
It appears that the modules defined here:
https://github.com/mozilla/lookml-generator/tree/main/generator/explores

Correspond to explore types? Could be wrong here.

Reply:
I can try running generator after merge and worse case scenario remove the explores here and add them in looker-spoke

Reply:
I see, thanks for pointing this out. I'll just take it out from here and add it to looker-spoke instead.